sql_generators/active_users_aggregates_v3/templates/mobile_schema.yaml

```diff
@@ -2,84 +2,92 @@ fields:
 - name: segment
   type: STRING
   mode: NULLABLE
-- name: app_version
-  type: STRING
-  mode: NULLABLE
-- name: attribution_medium
+  description: Classification of client_ids based on usage and active state.
+- name: app_name
   type: STRING
   mode: NULLABLE
-- name: attribution_source
+  description: Browser name.
+- name: app_version
   type: STRING
   mode: NULLABLE
-- name: attributed
```

Comment:
attribution_medium, attribution_source, and attributed all seem to be removed from the schema. Is that intentional?

Reply:
Not intentional, added back, thanks!